src/core/trulens/core/database/sqlalchemy.py

```diff
@@ -119,13 +119,15 @@ def __init__(
             )
 
     def _reload_engine(self):
-        self.engine = sa.create_engine(**self.engine_params)
+        if self.engine is None:
```

Comment:
Is this a bug in existing code?

Reply:
No, just don't want to create the engine from params if the user is passing it in instead.